test/integration/instance.test.js

```diff
@@ -2076,6 +2076,69 @@ describe(Support.getTestDialectTeaser('Instance'), () => {
     });
   });
 
+  describe('isSoftDeleted', () => {
```

Comment:
According to the linter in sequelize this is incorrect. It asks that you use arrow functions when possible over `function` and only use `function` when scope of `this` is a concern. 

`Unexpected function expression. (prefer-arrow-callback)`.

See https://github.com/sequelize/sequelize/blob/master/.eslintrc.json#L15